code/modules/hydroponics/grown/banana.dm

```diff
@@ -115,6 +116,8 @@
 /obj/item/weapon/grown/bananapeel/bluespace/Crossed(AM)
 	if(..())
 		var/teleport_radius = potency / 10
+		if(teleport_radius < 1) //not enough potency to teleport
+			return
 		do_teleport(AM, get_turf(AM), teleport_radius)
```

Comment:
this isn't a feature @xxalpha because if you get stuck in an endless teleport loop it kills the MC